LC1/src/main/res/values/strings.xml

```diff
@@ -34,6 +34,7 @@
     <string name="pref_zone_2">Zone 2 Name</string>
     <string name="pref_zone_3">Zone 3 Name</string>
     <string name="pref_zone_4">Zone 4 Name</string>
+    <string name="ssid_name">ivegotinternet24</string>
```

Comment:
Yeah I was attempting to tell the watch of the SSID that the wifi bridge was on, but never got it working properly, so had to hard code my SSID in there :)